src/ADAL.PCL/RequestData.cs

```diff
@@ -47,5 +47,8 @@ class RequestData
 
         public bool ExtendedLifeTimeEnabled { get; set; }
 
+        public string RequestId { get; set; }
+
+        public Guid CorrelationId { get; set; }
```

Comment:
>CorrelationId [](start = 20, length = 13)

correlation ID is not used anywhere. It is also already present or created in Authenticator.